src/requests/workspace.jl

```diff
@@ -73,55 +73,53 @@ function process(r::JSONRPC.Request{Val{Symbol("workspace/didChangeConfiguration
     request_julia_config(server)
 end
 
-function request_julia_config(server)
+function request_julia_config(server::LanguageServerInstance)
     response = JSONRPCEndpoints.send_request(server.jr_endpoint, "workspace/configuration", ConfigurationParams([
-        (ConfigurationItem(missing, "julia.format.$opt") for opt in fieldnames(DocumentFormat.FormatOptions))...;
-        ConfigurationItem(missing, "julia.lint.run");
-        (ConfigurationItem(missing, "julia.lint.$opt") for opt in fieldnames(StaticLint.LintOptions))...
+        ConfigurationItem(missing, "julia.format.indent"), # FormatOptions
+        ConfigurationItem(missing, "julia.format.indents"),
+        ConfigurationItem(missing, "julia.format.ops"),
+        ConfigurationItem(missing, "julia.format.tuples"),
+        ConfigurationItem(missing, "julia.format.curly"), 
+        ConfigurationItem(missing, "julia.format.call"),
+        ConfigurationItem(missing, "julia.format.iterOps"),
+        ConfigurationItem(missing, "julia.format.comments"),
+        ConfigurationItem(missing, "julia.format.docs"),
+        ConfigurationItem(missing, "julia.format.lineends"),
+        ConfigurationItem(missing, "julia.format.kw"),
+        ConfigurationItem(missing, "julia.lint.call"), # LintOptions
+        ConfigurationItem(missing, "julia.lint.iter"),
+        ConfigurationItem(missing, "julia.lint.nothingcomp"),
+        ConfigurationItem(missing, "julia.lint.constif"),
+        ConfigurationItem(missing, "julia.lint.lazyif"),
+        ConfigurationItem(missing, "julia.lint.datadecl"),
+        ConfigurationItem(missing, "julia.lint.typeparam"),
+        ConfigurationItem(missing, "julia.lint.modname"),
+        ConfigurationItem(missing, "julia.lint.pirates"),
+        ConfigurationItem(missing, "julia.lint.useoffuncargs"),
+        ConfigurationItem(missing, "julia.lint.run"),
+        ConfigurationItem(missing, "julia.lint.missingrefs")
         ]))
+
+    new_DF_opts = DocumentFormat.FormatOptions([isnothing(opt) ? DocumentFormat.default_options[i] : opt for (i,opt) in enumerate(response[1:11])]...)
```

Comment:
The point of the constructors in DocumentFormat/StaticLint was so that doing these kind of checks for nothingness here is unnecessary. Is the concern that the language server receives non-Bool, non-nothing responses?

```suggestion
new_DF_opts = DocumentFormat.FormatOptions(response[1:11]...)
```

Reply:
Sorry, I absolutely missed that - was a bit of a rush job. Could you make a PR to revert this?

Reply:
Sure thing.